saltproc/saltproc.py

```diff
@@ -362,8 +369,10 @@ def reopen_db(self, restart):
         self.waste_tank_db = self.f['waste tank composition']
         self.isolib_db = self.f['iso names']
         self.fissile_tank_db = self.f['fissile tank composition']
+        self.isozai_db = self.f['iso zai']
 
         if restart:
+            # resize dataset
```

Comment:
If it needs a comment to be readable, then this should probably be its own function.